src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -79,7 +79,7 @@
 )
 from zenml.io import fileio
 from zenml.logger import get_logger
-from zenml.model.model_stages import ModelStages
+from zenml.model import ModelStages
```

Comment:
Was this changed intentionally?

```suggestion
from zenml.model. model_stages import ModelStages
```